sp041_20191205T1757-instrument-scalar_v2.cdl

```diff
@@ -0,0 +1,417 @@
+      LONGITUDE:interpolation_methodology_vocabulary = "TBD" ;
+      LONGITUDE:interpolation_methodology_doi = "TBD" ;
+	float PRES(N_MEASUREMENTS) ;
+		PRES:_FillValue = -9999.9 ;
+		PRES:long_name = "Pressure" ;
+		PRES:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/PRES/";
+		PRES:standard_name = "sea_water_pressure" ;
+		PRES:units = "dbar" ;
+		PRES:ancillary_variables = "PRES_QC" ;
+		PRES:comment = "Sea water pressure, equals 0 at sea-level" ;
+        PRES:sensor = "/CTD";
+	byte PRES_QC(N_MEASUREMENTS) ;
+		PRES_QC:_FillValue = 0b ;
+		PRES_QC:long_name = "quality flag" ;
+	float DEPTH(N_MEASUREMENTS) ;
+		DEPTH:_FillValue = NaN ;
+		DEPTH:long_name = "Depth" ;
+		DEPTH:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/DEPTH/";
+		DEPTH:standard_name = "depth" ;
+		DEPTH:units = "m" ;
+		DEPTH:ancillary_variables = "DEPTH_QC" ;
+		DEPTH:positive = "down" ;
+        DEPTH:sensor = "/CTD";
+	byte DEPTH_QC(N_MEASUREMENTS) ;
+		DEPTH_QC:_FillValue = 0b ;
+		DEPTH_QC:long_name = "quality flag" ;
+	float TEMP(N_MEASUREMENTS) ;
+		TEMP:_FillValue = -9999.9 ;
+		TEMP:long_name = "Sea Water Temperature" ;
+        TEMP:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/TEMP/";
+		TEMP:standard_name = "sea_water_temperature" ;
+		TEMP:units = "Celsius" ;
+		TEMP:ancillary_variables = "TEMP_QC" ;
+		TEMP:valid_max = 40. ;
+		TEMP:valid_min = -5. ;
+		TEMP:coverage_content_type = "physicalMeasurement" ;
+		TEMP:coordinates = "time lon lat depth" ;
+		TEMP:comment = "Sea temperature in-situ ITS-90 scale" ;
+        TEMP:sensor = "/CTD";
+	byte TEMP_QC(N_MEASUREMENTS) ;
+		TEMP_QC:_FillValue = 0b ;
+		TEMP_QC:long_name = "quality flag" ;
+	float PSAL(N_MEASUREMENTS) ;
+		PSAL:_FillValue = NaN ;
+		PSAL:long_name = "Sea Water Salinity" ;
+        PSAL:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/PSAL/";
+		PSAL:standard_name = "sea_water_practical_salinity" ;
+		PSAL:units = "1" ;
+		PSAL:ancillary_variables = "PSAL_QC" ;
+		PSAL:valid_max = 40. ;
+		PSAL:valid_min = 0. ;
+		PSAL:coverage_content_type = "physicalMeasurement" ;
+		PSAL:coordinates = "time lon lat depth" ;
+		PSAL:comment = "Practical salinity computed using UNESCO 1983 algorithm" ;
+        PSAL:sensor = "/CTD";
+	byte PSAL_QC(N_MEASUREMENTS) ;
+		PSAL_QC:_FillValue = 0b ;
+		PSAL_QC:long_name = "quality flag" ;
+	float CHLA(N_MEASUREMENTS) ;
+		CHLA:_FillValue = NaN ;
+		CHLA:long_name = "Chlorophyll-a concentration" ;
+        CHLA:vocabulary = "http://vocab.nerc.ac.uk/collection/OG1/current/CHLA/";
+		CHLA:standard_name = "mass_concentration_of_chlorophyll_a_in_sea_water" ;
+		CHLA:units = "mg m-3" ;
+		CHLA:ancillary_variables = "CHLA_QC" ;
+		CHLA:coverage_content_type = "physicalMeasurement" ;
+		CHLA:coordinates = "time lon lat depth" ;
+		CHLA:comment = "In-situ fluorometer with either manufacturer, laboratory or sample calibration applied" ;
+        CHLA:sensor = "/FLUOROMETER_CHLA";
```

Comment:
This currently points to the old CHLA group. Should point to the INSTRUMENT_CHLA variable, i.e.: CHLA:sensor = "INSTRUMENT_CHLA".

Reply:
document updated accordingly